.github/workflows/release.yaml

```diff
@@ -127,13 +127,15 @@ jobs:
       upload-assets: true
 
   generate-sbom:
-    name: Create Sbom and sign assets
+    name: Create SBOM and generate hash
     needs:
       - argocd-image
       - goreleaser
     permissions:
       contents: write # Needed for release uploads
       id-token: write # Needed for signing Sbom
```

Comment:
We can remove `id-token: write` permission as this will no longer be needed.

Reply:
Great catch! Thank you! Changed title as well.

Reply:
My bad, gave a horrible example of title description.    `chore` should be lowercase, sorry about that.

Reply:
No worries!